dist/src/main/resources/api/cluster/cluster-api.yaml

```diff
@@ -103,6 +103,34 @@ paths:
         '504':
           $ref: 'components.yaml#/responses/TimeoutError'
 
+    patch:
+      summary: Reconfigure cluster
+      description: Reconfigure cluster by adding or removing brokers, adding more partitions, or
+                   changing the replicationFactor. Note that adding partitions is an experimental
+                   feature and not fully supported.
```

Comment:
:upside_down_face: This is deeply indented. Above you have changed another description to be less indented. Not sure how this will look in docs etc.

Reply:
Not sure though. It was auto-formatted. :thinking: